src/Misc/layoutroot/run.cmd

```diff
@@ -13,21 +13,43 @@ if defined VERBOSE_ARG (
 rem Unblock files in the root of the layout folder. E.g. .cmd files.
 powershell.exe -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "$VerbosePreference = %VERBOSE_ARG% ; Get-ChildItem -LiteralPath '%~dp0' | ForEach-Object { Write-Verbose ('Unblock: {0}' -f $_.FullName) ; $_ } | Unblock-File | Out-Null"
 
-if /i "%~1" equ "localRun" (
-    rem ********************************************************************************
-    rem Local run.
-    rem ********************************************************************************
-    "%~dp0bin\Runner.Listener.exe" %*
-) else (
-  rem ********************************************************************************
-  rem Run.
-  rem ********************************************************************************
-  "%~dp0bin\Runner.Listener.exe" run %*
-
-  rem Return code 4 means the run once runner received an update message.
-  rem Sleep 5 seconds to wait for the update process finish and run the runner again.
-  if ERRORLEVEL 4 (
-    timeout /t 5 /nobreak > NUL
-    "%~dp0bin\Runner.Listener.exe" run %*
-  )
+
```

Comment:
regarding the ability to update run.sh/run.cmd....

i wonder whether we should put the bulk of the contents in a separate script and then run.cmd effectively does nothing but `call bin/run-helper.cmd` (in a loop?)

Reply:
maybe you end up with the same problem, idk

Reply:
Done that. 

`run.sh|cmd` is now very small and is almost only responsible for running `run-helper.sh|cmd`.

`run.sh|cmd` will potentially be very long running processes, and as such, hard for us to change: changing a running `.cmd` is as [unpredictable](https://stackoverflow.com/questions/906586/changing-a-batch-file-when-its-running) as changing a running [shell script](https://stackoverflow.com/questions/3398258/edit-shell-script-while-its-running)

Before merging this change, we should consider coming up with a way to service these files if we absolutely need to, perhaps an update could force a restart of `run.sh|cmd` through `update.sh`.

Thoughts? @ericsciple @TingluoHuang